.github/workflows/main.yml

```diff
@@ -90,7 +90,10 @@ jobs:
           - TEST=HDFDataset
           - TEST=TranslationDataset
           - TEST=LearningRateControl
+          - TEST=Log
+          - TEST=Pretrain
           - TEST=SprintDataset
+          - TEST=SprintInterface
```

Comment:
This test requires TF, right?

Reply:
In this case we can move it, I already thought of doing that but then went with base-tests because I thought this should work either way (the test used Theano before, but could be re-written for TF, so it is a rather independent feature that is tested here).

Reply:
But you did exactly that, or not? I.e. you just rewrote it that it depends on TF?

Reply:
Yes sure. But we never defined if the tf-tests are "Tests that are testing TF specific features" or "Any test that depends somehow on TF". From your comments it now is clear you want it structured in the second way, so I will oblige to that.

Reply:
No I did not say that. I did not say I want to have it structured this way.

This is just the only difference of the current structure in the CI tests, that one have a TF dependency and one have not. In no way does it define what kinds of tests are in what structure, except of this aspect what dependencies they need, because that is how the structures differ.

If you really want, you can create another section for tests which have a TF dependency but are not really TF specific. But I'm not really sure this adds anything?